### ISSUE TITLE
question on multiple agents

### ISSUE THREAD

**Issue by mattinjersey** [Unity-Technologies/ml-agents#2212]
I would like to build a fighting game with 2 agents. They both would use the same input data, which would be the image of the screen.
Does each agent brain have to send the scene image ? This would mean that we are sending the same data twice.

**Comment by roboserg** [comment]
if the observations are identical for both agents then you can just use two, identical brains (two instances of the same agent with the same brain).

**Comment by mattinjersey** [comment]
what if they were not identical characters for example a bunny and a wolf but they both use the same image as data

**Comment by roboserg** [comment]
ok let me rephrase it, is the _observations_ and _actions_ are identical, you may use the same brain.

**Comment by xiaomaogy** [comment]
@mattinjersey If you have two different characters, then you would need two different models to control them. So during training you would need to feed the picture to both of these models. I agree it is sending the data twice.

**Comment by ervteng** [comment]
Thanks for the comments - closing this issue due to inactivity. Feel free to open a new issue if you're still running into problems.
